src/pylife/materiallaws/woehlercurve.py

```diff
@@ -52,10 +52,6 @@ class WoehlerCurve(PylifeSignal):
                or calculated from ``TN`` if given.
     """
 
-    def __init__(self, pandas_obj):
-        self._obj = pandas_obj.copy()
-        self._validate()
```

Comment:
But what is the difference now? The exact same constructor is in `PylifeSignal` so nothing changed?

Reply:
ah except the copy